src/Keyboard.js

```diff
@@ -0,0 +1,239 @@
+export default class Keyboard {
+  constructor() {
+    this.symbols = [
+      [
+        ['`~ёЁ', 'Backquote'],
+        ['1!1!', 'Digit1'],
+        ['2@2"', 'Digit2'],
+        ['3#3№', 'Digit3'],
+        ['4$4;', 'Digit4'],
+        ['5%5%', 'Digit5'],
+        ['6^6:', 'Digit6'],
+        ['7&7&', 'Digit7'],
+        ['8*8*', 'Digit8'],
+        ['9(9(', 'Digit9'],
+        ['0)0)', 'Digit0'],
+        ['-_-_', 'Minus'],
+        ['=+=+', 'Equal'],
+        ['', 'Backspace'],
+      ],
+      [
+        ['', 'Tab'],
+        ['qQйЙ', 'KeyQ'],
+        ['wWцЦ', 'KeyW'],
+        ['eEуУ', 'KeyE'],
+        ['rRкК', 'KeyR'],
+        ['tTеЕ', 'KeyT'],
+        ['yYнН', 'KeyY'],
+        ['uUгГ', 'KeyU'],
+        ['iIшШ', 'KeyI'],
+        ['oOщЩ', 'KeyO'],
+        ['pPзЗ', 'KeyP'],
+        ['[{хХ', 'BracketLeft'],
+        [']}ъЪ', 'BracketRight'],
+        ['\\|\\/', 'Backslash'],
+      ],
+      [
+        ['', 'CapsLock'],
+        ['aAфФ', 'KeyA'],
+        ['sSыЫ', 'KeyS'],
+        ['dDвВ', 'KeyD'],
+        ['fFаА', 'KeyF'],
+        ['gGпП', 'KeyG'],
+        ['hHрР', 'KeyH'],
+        ['jJоО', 'KeyJ'],
+        ['kKлЛ', 'KeyK'],
+        ['lLдД', 'KeyL'],
+        [';:жЖ', 'Semicolon'],
+        ["'\"эЭ", 'Quote'],
+        ['', 'Enter'],
+      ],
+      [
+        ['', 'ShiftLeft'],
+        ['zZяЯ', 'KeyZ'],
+        ['xXчЧ', 'KeyX'],
+        ['cCсС', 'KeyC'],
+        ['vVмМ', 'KeyV'],
+        ['bBиИ', 'KeyB'],
+        ['nNтТ', 'KeyN'],
+        ['mMьЬ', 'KeyM'],
+        [',<бБ', 'Comma'],
+        ['.>юЮ', 'Period'],
+        ['/?.,', 'Slash'],
+        ['', 'ShiftRight'],
+      ],
+      [
+        ['', 'ControlLeft'],
+        ['', 'AltLeft'],
+        ['', 'Space'],
+        ['', 'AltRight'],
+        ['', 'ControlRight'],
+      ],
+    ];
```

Comment:
Good for memory, but hard to understand easy. You should rework structure to make it more readable to new developers, that will be involved in your project.